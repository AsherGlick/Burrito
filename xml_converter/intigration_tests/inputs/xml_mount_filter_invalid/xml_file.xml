<OverlayData>
<MarkerCategory  Name="mycategory">
</MarkerCategory>

<POIs>
<POI Mount="" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" IconFile="texture.png" Type="mycategory"/>
<POI Mount="NotAMount" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" IconFile="texture.png" Type="mycategory"/>
<POI Mount="Raptor,Springer," Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" IconFile="texture.png" Type="mycategory"/>
<POI Mount="Raptor,NotAMount,Springer" Type="mycategory" XPos="169.81" YPos="210.65" ZPos="215.83" MapID="50" IconFile="texture.png" Type="mycategory"/>
</POIs>
</OverlayData>
