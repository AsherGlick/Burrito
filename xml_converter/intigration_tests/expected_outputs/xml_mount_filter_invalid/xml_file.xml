<OverlayData>
<MarkerCategory  Name="mycategory">
</MarkerCategory>

<POIs>
<POI  Type="mycategory" IconFile="texture.png" MapID="50" Mount="" XPos="169.809998" YPos="210.649994" ZPos="215.830002"/>
<POI  Type="mycategory" IconFile="texture.png" MapID="50" Mount="" XPos="169.809998" YPos="210.649994" ZPos="215.830002"/>
<POI  Type="mycategory" IconFile="texture.png" MapID="50" Mount="raptor,springer" XPos="169.809998" YPos="210.649994" ZPos="215.830002"/>
</POIs>
</OverlayData>
